assets/examples/50_delayed_connections.iesopt.yaml

```
@@ -0,0 +1,44 @@
+config:
+  general:
+    version:
+      core: 2.4.0
+  optimization:
+    problem_type: LP
+    solver:
+      name: highs
+    snapshots:
+      count: 4
+      weights: 0.25
+  results:
+    enabled: true
+    memory_only: true
+
+carriers:
+  water: {}
+
+components:
+  upper:
+    type: Node
+    carrier: water
+  lower:
+    type: Node
+    carrier: water
+  river:
+    type: Connection
+    node_from: upper
+    node_to: lower
+    delay: 0.33
```

```suggestion
    delay: 1.0 / 3.0
```

or something like that to not incentivize people using the example to use `0.33` for "a third of an hour" and then being surprised by some small "rounding error" in the results?

---

good point! done